src/Engine/ProtoCore/Namespace/ElementRewriter.cs

```diff
@@ -11,8 +11,15 @@
 namespace ProtoCore.Namespace
 {
 
-    public static class ElementRewriter
+    public class ElementRewriter
```

Comment:
Yes this class basically has a single public API but is there to encapsulate all AST rewrite related helper sub-routines (that are private).